### ISSUE TITLE
Is it possible to customize our own exploiters

### ISSUE THREAD

**Issue by Colstuwjx** [guardicore/monkey#4128]
## Documentation request

**What do you think should be changed?**
I've seen we've predefined a several [exploiters](https://techdocs.akamai.com/infection-monkey/docs/exploiters), however, sometimes it may doesn't match the real world requirement, e.g. Redis or Docker has published a CVE which could cause a security issue, but we can't use infection monkey to test these kind of scenarios.

Is it possible to customize our own exploiters so that we can test more kind of CVEs or potential risk cases for our own requirements.

**Comment by mssalvatore** [comment]
Yes, it is possible. Unfortunately, at the present time the process for building plugins is not documented. The focus of our next release (expected mid-July) is to publish a plugin SDK and documentation to enable users to build their own plugins.

Between now and then, we'd be happy to get you set up and walk you through the process. Please [join our slack workspace](https://join.slack.com/t/infectionmonkey/shared_invite/zt-2cm5qiayf-yiEg5RPau0zQhki9xTlORA) and drop a comment in #monkey_development. Someone will be happy to help you out.
